setup.py

```diff
@@ -0,0 +1,61 @@
+from setuptools import setup
+
+filename = 'rfhub2/version.py'
+exec(open(filename).read())
+
+setup(
+    name='rfhub2',
+    version=version,
+    author=['Pawel Bylicki', 'Maciej Wiczk'],
+    author_email=['pawelkbylicki@gmail.com', 'maciejwiczk@gmail.com'],
+    url='https://github.com/pbylicki/rfhub2/',
+    keywords='robotframework',
+    license='Apache License 2.0',
+    description='Webserver for robot framework and python assets documentation',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    zip_safe=True,
+    include_package_data=True,
+    install_requires=['aiofiles==0.4.0', 'Click==7.0', 'dataclasses==0.6', 'fastapi==0.30.0', 'h11==0.8.1',
+                      'httptools==0.0.13', 'jinja2==2.10.1', 'MarkupSafe==1.1.1', 'pydantic==0.28',
+                      'robotframework==3.1.2', 'SQLAlchemy==1.3.3', 'starlette==0.12.0', 'uvicorn==0.7.1',
+                      'uvloop==0.12.2', 'websockets==7.0', 'requests==2.22.0'],
+    extras_require={
+        "postgresql": ["psycopg2-binary==2.8.3"]
+    },
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: Unix",
+        "Framework :: Robot Framework",
+        "Framework :: Robot Framework :: Tool",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Software Development :: Testing",
+        "Topic :: Software Development :: Quality Assurance",
+        "Intended Audience :: Developers",
+    ],
+    packages=[
+        'rfhub2',
+        'rfhub2.api',
+        'rfhub2.api.endpoints',
+        'rfhub2.api.middleware',
+        'rfhub2.api.utils',
+        'rfhub2.cli',
+        'rfhub2.db',
+        'rfhub2.db.model',
+        'rfhub2.db.repository',
+        'rfhub2.model',
+        'rfhub2.ui',
+        'rfhub2.utils',
+    ],
+    scripts=[],
+    entry_points={
+        'console_scripts': [
+            "rfhub2 = rfhub2.__main__:main",
+            "rfhub2.cli = rfhub2.cli.__main__:main",
```

Comment:
I was wondering if we can make this executable name more compact and still recognizable, but i couldn't come up with anything. I was just thinking that maybe in could be switched to `rfhub2-cli` to not mistake cli for file extension

Reply:
Sure thing!